MonoGame.Framework/Platform/Input/GamePad.iOS.cs

```diff
@@ -151,10 +151,16 @@ private static GamePadState PlatformGetState(int index, GamePadDeadZone leftDead
                     if (controller.ExtendedGamepad.RightTrigger.IsPressed)
                         buttons |= Buttons.RightTrigger;
 
-                    if (controller.ExtendedGamepad.ButtonMenu.IsPressed)
+                    if (controller.ExtendedGamepad.ButtonMenu != null
+                    && controller.ExtendedGamepad.ButtonMenu.IsPressed)
```

Comment:
If you feel like it, you may write this as ```controller.ExtendedGamepad.ButtonMenu?.IsPressed```. Since the iOS specific code doesn't have to abide to the C# 5 rule, it's fine in this occurrence and won't fail ```ConsoleCheck```.

Reply:
@ThomasFOG the issue we have is that the Xamarin bindings have some properties as nullable, like `controller.ExtendedGamepad.ButtonOptions?`, while others are not. Hence why I  used the above.